modules/git/commit.go

```diff
@@ -474,7 +474,7 @@ func (c *Commit) GetBranchName() (string, error) {
 	}
 
 	// name-rev commitID output will be "COMMIT_ID master" or "COMMIT_ID master~12"
```

Comment:
```suggestion
	// name-rev commitID output will be "COMMIT_ID master" or "COMMIT_ID pull/5/head"
```
remove the ~ example ... and add pull example

Reply:
Hmm, this comment is right, only last commit of a branch is not have number in output ....

Reply:
we can also just remove the ~... example?

Reply:
It's right. why remove? 🤔
![image](https://user-images.githubusercontent.com/25342410/81028520-62bc8180-8eb4-11ea-99d9-f137f2da669c.png)